org.scala-ide.sdt.core.tests/src/org/scalaide/core/testsetup/SDTTestUtils.scala

```diff
@@ -67,13 +67,21 @@ object SDTTestUtils extends HasLogger {
   def findProblemMarkers(unit: ICompilationUnit) =
     unit.getUnderlyingResource().findMarkers(IJavaModelMarker.JAVA_MODEL_PROBLEM_MARKER, true, IResource.DEPTH_INFINITE)
 
+  def findProjectProblemMarkers(project: IProject, types: String*) = {
+    (for {
+      typ <- types
+    } yield project.findMarkers(typ, false, IResource.DEPTH_INFINITE).toList).flatten
+  }
+
+  def userFriendlyMarkers(markers: List[IMarker]) = markers.map(_.getAttribute(IMarker.MESSAGE))
```

Comment:
I think the name is not informative enough. It should be something like `markerMessage` or something along those lines (I know this might be an old method, it was wrong in the first place :)).